scripts/install/default-env-vars.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export PANDIR=/var/panoptes
+export PANLOG=${PANDIR}/logs
+export POCS=${PANDIR}/POCS
+export PAWS=${PANDIR}/PAWS
+export PIAA=${PANDIR}/PIAA
+export MPLBACKEND=${POCS}/.matplotlib
```

Comment:
I'm unfamiliar with this file. I don't see a .matplotlib in my clone.

Reply:
Sorry, meant to make a comment that this should go after #423, where I added that.